src/common/talents/TALENTS_DEMON_HUNTER.js

```diff
@@ -3,7 +3,7 @@
 export default {
   // Shared
   FELBLADE_TALENT: { id: 232893, name: "Felblade", icon: "ability_demonhunter_felblade" },
-  FEL_ERUPTION_TALENT: { id: 211881, name: "Fel Eruption", icon: "ability_bossfellord_felspike" },
+  FEL_ERUPTION_TALENT: { id: 211881, name: "Fel Eruption", icon: "ability_bossfellord_felspike", pain: 10},
```

Comment:
These files are auto generated so this will get overwritten, don't do this! I'll add pain to the auto generation, so you should revert these changes and see it appear in the master branch shortly.